src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +53,12 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        Predicate<Person> predicate = person -> person.getAge() >= fromAge
```

Comment:
Let's define concrete naming.
```suggestion
        Predicate<Person> applicableMenPredicate = person -> person.getAge() >= fromAge
```